cmake/modules/CompilerFlags.cmake

```diff
@@ -41,7 +41,7 @@ if(NOT WIN32)
 else()
 	set(MINIMAL_BUILD ON)
 
-	set(SYSDIG_COMMON_FLAGS "-D_CRT_SECURE_NO_WARNINGS -DWIN32 -DMINIMAL_BUILD /EHsc /W3 /Zi")
+	set(SYSDIG_COMMON_FLAGS "-D_CRT_SECURE_NO_WARNINGS -DWIN32 -DMINIMAL_BUILD /EHsc /W3 /Zi /std:c++latest")
```

Comment:
Can we stick to C++11 (same as the Linux/Mac builds)?

Reply:
According to https://docs.microsoft.com/en-us/cpp/build/reference/std-specify-language-standard-version?view=msvc-140, I don't see a way to only use c++11. c++14 is the default, though, so I'm going to remove this entirely.